projects/ad9208/src/main.c

```diff
@@ -488,6 +492,26 @@ int main(void)
 	if (status != SUCCESS)
 		xil_printf("axi_dmac_transfer() error: %"PRIi32"\n", status);
 
+#ifdef IIO_SUPPORT
+	printf("The board accepts libiio clients connections through the serial backend.\n");
+
+	struct iio_axi_adc_init_param iio_axi_adc_0_init_par;
+	iio_axi_adc_0_init_par = (struct iio_axi_adc_init_param) {
+		.rx_adc = rx_0_adc,
+		.rx_dmac = rx_dmac,
+		.adc_ddr_base = DDR_MEM_BASEADDR + 0x800000,
```

Comment:
Will using the same `adc_ddr_base` for both ADCs not overwrite the data from one of them in case they sample simultaneously? In short, can they both sample at the same time in this configuration?

Reply:
![image](https://user-images.githubusercontent.com/26061529/87405688-75e08180-c5c8-11ea-92fb-23ae008ee555.png)

Red trace is channel 1 from the first adc
Orange trace is channel 1 from the second adc
Both adc are running in the ramp test mode.
As you may notice, the samples of the two channels are interspersed.